src/components/MediaCounterButton/MediaCounterButton.story.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withDocs } from 'storybook-readme';
+import { select, number } from '@storybook/addon-knobs';
+
+import MediaCounterButton from './';
+import README from './README.md';
+
+storiesOf('Components/MediaCounterButton', module)
+  .addDecorator(withDocs(README))
+  .add('default', () => (
+    <MediaCounterButton
+      variant={select('Variant', ['dark', 'light'], 'dark')}
+      totalImages={number('Total Images', 5)}
+      onClick={() => {
+        console.log('Image counter button has been clicked');
+      }}
```

Comment:
With this old story format we can use `addon-actions` in the legacy way: https://github.com/storybookjs/storybook/blob/v6.3.8/addons/actions/ADVANCED.md

```suggestion
      onClick={action('onClick')}
```

Reply:
I made the change locally and it didn't seem to work (likely because of the story format), so it's not as useful _now_ but will become more useful when we upgrade all stories to the new format. Although we could change this story to the new format, but that's out of scope for this PR.